src/modules/app/routes/landing.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import { citiesFetcher } from '../../endpoint/fetchers'
+import { createAction } from 'redux-actions'
+
+import type { Dispatch, GetState } from 'redux-first-router/dist/flow-types'
+import { clearStoreOnCityChange } from '../../endpoint/actions/remover'
+
+export const LANDING_ROUTE = 'LANDING'
+
+export const goToLanding = (language: string) => createAction(LANDING_ROUTE)({language})
+
+/**
+ * LandingRoute, matches /landing/de
+ * @type {{path: string, thunk: function(Dispatch, GetState)}}
+ */
+export const landingRoute = {
+  path: '/landing/:language',
```

Comment:
So is '/' no longer the landing page? I'm also a bit agains introducting a new route if we already have one. We should keep the set of routes minimal.

Reply:
it also wasn't the landing page before, / was the i18nRedirect

Reply:
the landing was added because it's easier to differ between a city name and a language code, because there are also city names which are only three digits long (we don't have one right now though)